ramls/pick-slips-response.json

```diff
@@ -4,69 +4,198 @@
   "type": "object",
   "properties": {
     "pickSlips": {
-      "description": "A collection of current pick slips",
+      "description": "Collection of current pick slips",
       "id": "pickSlips",
       "type": "array",
       "items": {
-        "title": "A pick slip",
-        "description": "A pick slip representation of an item",
+        "title": "Pick slip",
+        "description": "Pick slip representation of item, request and requester",
         "type": "object",
         "properties": {
-          "id": {
-            "description": "ID of the item",
-            "type": "string",
-            "$ref": "uuid.json"
-          },
-          "title": {
-            "description": "The title of the item",
-            "type": "string"
-          },
-          "callNumber" : {
-            "description" : "The effective call number of the item",
-            "type": "string"
-          },
-          "holdingsRecordId": {
-            "description": "The ID of the holdings for the item",
-            "type": "string",
-            "$ref": "uuid.json"
-          },
-          "instanceId": {
-            "description": "The ID of the instance for the item",
-            "type": "string",
-            "$ref": "uuid.json"
-          },
-          "barcode": {
-            "description": "The barcode of the item",
-            "type": "string"
-          },
-          "status": {
-            "description": "The status of the item",
-            "type": "string"
+          "item": {
```

Comment:
This appears to be a breaking compatibility change to the public API of the module. Is that the case?